Tests/SodiumTests/ReadmeTests.swift

```diff
@@ -28,7 +28,7 @@ class ReadmeTests : XCTestCase {
     ]
 
     func testAuthenticatedEncryption() {
-        let sodium = Sodium()
+        let sodium = try! Sodium()
```

Comment:
That change is probably breaking all the example applications as well.

Reply:
My bad. Fixed